### ISSUE TITLE
Filtered room is filled with my username when I go back from the page Settings

### ISSUE THREAD

**Issue by antoyo** [hydrogen-sailfishos/sfos-hydrogen#41]
Hi.
The following actions will fill the "Filter rooms..." input text with my username:

 * Click on the Settings icon.
 * Click on back to go back to the list of conversations.

I would expect the input text to not be filled.

I don't know if this is an issue with upstream of the SailfishOS port.

Thanks.

**Comment by b100dian** [comment]
Hi, thank you for your bug report.
I recall to have noticed this while at the November hackathon, however I do not see it now, not on my device and nor on https://hydrogen.element.io which also runs 0.4.1 today.
Are you able to reproduce it on a desktop browser too?

**Comment by antoyo** [comment]
No. I cannot reproduce in my desktop browser.
Should I try on Firefox or Chrome to mimic most closely the app?

I have hydrogen-sailfishos 0.4.1+master on a Sony Xperia XA2 with SailfishOS version 4.5.0.24.

**Comment by b100dian** [comment]
I think Firefox with a very narrow window..
But thanks for stating that you're running the latest, I am sure we'll find a way to repro it again - @pherjung maybe you have seen it too? If not, we can check the upstream source.
Initial grepping doesn't show much on "room-filter" other than the fact that [autocomplete](https://github.com/element-hq/hydrogen-web/blob/master/src/platform/web/ui/session/leftpanel/LeftPanelView.js#L79) is on. Maybe this happens only in the session you initially log in, and username is retained by the browser?

**Comment by antoyo** [comment]
The issue doesn't happen in Firefox either. Any other thing I could test?

**Comment by pherjung** [comment]
Yes, I already noticed this issue. I don't know where is the bug, but if I find something I'll comment here.

**Comment by yajo10** [comment]
It suddenly is reproduce able for me now as well.

**Comment by b100dian** [comment]
Is this with an user name (as in, lowercase etc.) or with a Name (which may have the first letter uppercased)? I ask to determine if there's a name clash with the login field (in which case one should remember if Save password was used) or just an application bug.

**Comment by pherjung** [comment]
Strangely, I haven't been able to figure out how to reproduce this bug. At the moment it looks really random.

**Comment by antoyo** [comment]
> Is this with an user name (as in, lowercase etc.) or with a Name (which may have the first letter uppercased)? I ask to determine if there's a name clash with the login field (in which case one should remember if Save password was used) or just an application bug.

It's the username (lowercase).

**Comment by ichthyosaurus** [comment]
I have multiple accounts and in my case it is actually the username (lowercase) of the first account: if my accounts are "jim", "john", "jane", then it is always "jim" no matter which account I used to open the settings page.

**Comment by ichthyosaurus** [comment]
This is https://github.com/element-hq/hydrogen-web/issues/380. I proposed a fix in https://github.com/element-hq/hydrogen-web/pull/1200.
